### PR TITLE
Game history

### PR DESCRIPTION
**Pull request by itsalaidbacklife**
Implement & style game logs/history

### CODE REVIEW

File: src/views/GameView.vue [hunk 1]
```diff
@@ -1071,6 +1075,12 @@ export default {
 		max-width: 80%;
 		height: 100%;
 	}
+	#history {
+		& #history-logs {
+			height: 80%;
+			overflow: scroll;
```
Comment (lihaobhsfer) [inline src/views/GameView.vue:1081]: I see a blank vertical and horizontal scroll bar again on my end. Not sure if this is just me. changing this to `auto` seems to fix it.

**Review by lihaobhsfer — CHANGES_REQUESTED** [review]
pls see comment above

File: src/views/GameView.vue (hunk 1 above)
Comment (itsalaidbacklife) [reply src/views/GameView.vue]: Set to overflow:auto. Can you confirm works on your machine?

Comment (lihaobhsfer) [reply src/views/GameView.vue]: Works now!

**Pull request merged**